Blish HUD/Program.cs

```diff
@@ -44,20 +46,23 @@ public static class Program {
 
             Logger.Debug("Launched from {launchDirectory} with args {launchOptions}.", Directory.GetCurrentDirectory(), string.Join(" ", args));
 
-            if (IsMoreThanOneInstance()) {
+            // Single instance handling
+            _singleInstanceMutex = new Mutex(true, ApplicationSettings.Instance.MumbleMapName != null
+                                                       ? $"{APP_GUID}:{ApplicationSettings.Instance.MumbleMapName}"
+                                                       : $"{APP_GUID}");
+
+            if (!_singleInstanceMutex.WaitOne(TimeSpan.Zero, true)) {
                 Logger.Warn("Blish HUD is already running!");
                 return;
             }
 
-            // Needed by textboxes to enable CTRL + A selection
-            Application.EnableVisualStyles();
```

Comment:
Why was this removed?

Reply:
It was unrelated, but noticed while updating the mutex stuff. 

This was needed by the old WinForms textbox which is no longer used.